src/components/TokenClaimDialog.tsx

```diff
@@ -187,19 +212,29 @@ export default function TokenClaimDialog({
               </Trans>
             </Typography>
           )}
-          {allPoolsWithRewards.map((pool, i, arr) => (
-            <React.Fragment key={pool.poolName}>
-              <ClaimListItem
-                title={pool.poolName}
-                amount={rewardBalances[pool.poolName] || Zero}
-                claimCallback={() => claimPoolReward(pool)}
-                status={
-                  claimsStatuses["allPools"] || claimsStatuses[pool.poolName]
-                }
-              />
-              {i < arr.length - 1 && <Divider key={i} />}
-            </React.Fragment>
-          ))}
+          {gauges.map(
+            (gauge, i, arr) =>
+              userState?.gaugeRewards?.[gauge?.address || ""]?.claimableSDL.gt(
+                Zero,
+              ) &&
+              gauge && (
+                <React.Fragment key={gauge?.poolName}>
+                  <ClaimListItem
+                    title={gauge?.poolName || ""}
+                    amount={
```

Comment:
seems easier to save this as a const since you're using it in two places within this block